tests/features/search.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Search
+	As a Devsecops Engineer
+	I want to perform searching across vulnerabilities, SBOMs and packages, specific searches for CVE IDs, SBOM titles, package names and show results that are easy to navigate to the specific item of interest.
+
+Background:
+	Given User is using an instance of the TPA Application
+	When User has successfully uploaded an SBOM
```

Comment:
imho `Background` should contain only `Given` steps - as in shared setUpClass call performed before each `Scenario` (or Outline instance), not When steps

at least per the docs - https://cucumber.io/docs/gherkin/reference/#background
although looking at code of playwright-bdd it seems it could handle here any type of step actually (not tested)

Given - actions to be performed as pre-test-setup
When - action to be performed as test (the actually tested action)
Then - assertion after/of the executed 'When' action(s)

Background section is meant for shared pre-setup initialization

Reply:
Added example - as Your first keyword is `Given` then it can be changed to followup `given`s by using `And`.